modules/tinymce/src/core/main/ts/keyboard/DeleteBackspaceKeys.ts

```diff
@@ -50,24 +51,54 @@ const executeKeydownOverride = (editor: Editor, caret: Cell<Text | null>, evt: K
   });
 };
 
-const executeKeyupOverride = (editor: Editor, evt: KeyboardEvent) => {
+const executeKeyupOverride = (editor: Editor, evt: KeyboardEvent, isBackspaceKeydown: boolean) => {
+  const platform = PlatformDetection.detect();
+  const os = platform.os;
+  const browser = platform.browser;
+  const multiDeleteKeyPatterns: MatchKeys.KeyPattern[] = os.isMacOS() ? [
+    { keyCode: VK.BACKSPACE, altKey: true, action: MatchKeys.action(InlineFormatDelete.refreshCaret, editor) },
+    { keyCode: VK.DELETE, altKey: true, action: MatchKeys.action(InlineFormatDelete.refreshCaret, editor) },
+  ] : [
+    { keyCode: VK.BACKSPACE, ctrlKey: true, action: MatchKeys.action(InlineFormatDelete.refreshCaret, editor) },
+    { keyCode: VK.DELETE, ctrlKey: true, action: MatchKeys.action(InlineFormatDelete.refreshCaret, editor) }
+  ];
+
+  // macOS surpresses keyup events for most keys including Backspace when Meta key is engaged
+  // To emulate Meta + Backspace on macOS, add a pattern for the meta key when backspace was
+  // detected on keydown
+  if (os.isMacOS() && isBackspaceKeydown) {
+    multiDeleteKeyPatterns.push({
+      // firefox detects Cmd as "Command" not "Meta"
+      keyCode: browser.isFirefox() ? 224 : 91,
```

Comment:
And there are no modern things we can use here like `.key` etc they are all incorrect? If firefox ever fixes this bug this code will break but not sure if we can avoid that we can't really feature detect that.

Reply:
`.key` shows `Meta` for both Firefox and Chrome/Safari. However, I think we are reliant on `keyCode` because that is what is accepted by the type `MatchKeys.KeyPattern[]` used in `MatchKeys.execute`. 

Not sure if it is safe to skip browser detection and just add patterns for both `keyCode: 224` and `keyCode: 91`.

Reply:
`key` is the long-term solution, I think there's a ticket in the backlog about what we need to switch to it but it has probably been shelved by now.